### PR TITLE
add links to mastodon including verification

### PR DESCRIPTION
**Pull request by ywwg**

### CODE REVIEW

**Comment by ywwg** [comment]
precommit failure looks unrelated to the change

File: theme/templates/base.html [hunk 1]
```diff
@@ -25,6 +25,9 @@
     {# should be equivalent to rel=canonical -- used for aggregating likes #}
     <meta property="article:publisher" content="https://www.facebook.com/pages/Mixxx-DJ-Software/21723485212"/>
 
+    <!-- Mastodon verification backlink -->
+    <a rel="me" href="https://floss.social/@mixxx">Mastodon</a>
+
```
Comment (Swiftb0y) [inline theme/templates/base.html]: Does this really have to go in the header? It creates an ugly artifact. Wouldn't it make more sense to add this as a separate entry in the Community links at the bottom? Or Just add `display: none` so it gets hidden. 
Screenshot from 2023-01-05 22-35-12

**Comment by Holzhaus** [comment]
Would be nice to add an icon + link to the header, next to the other social icons (Twitter, Facebook, Zulip, GitHub).

**Comment by Holzhaus** [comment]
An icon can be found here: https://commons.m.wikimedia.org/wiki/File:Mastodon_Logotype_(Simple).svg

Probably just need to change the color to white using Inkscape.

File: theme/templates/base.html (hunk 1 above)
Comment (ywwg) [reply theme/templates/base.html]: oh sorry, I thought it was a meta so it wouldn't appear.  I'll fix

**Review by Swiftb0y — APPROVED** [review]
Thanks LGTM. Objections @Holzhaus?

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you!

**Pull request merged**